tests/python/openassetio/pluginSystem/resources/pathA/modulePlugin.py

```diff
@@ -1,16 +1,18 @@
 """
-Provides a test PluginSystemPlugin implemented within a single file
+Provides a test PythonPluginSystemPlugin implemented within a single file
 module.
 """
 
-from openassetio.pluginSystem import PluginSystemPlugin
+from openassetio.pluginSystem import PythonPluginSystemPlugin
 
-class ModulePlugin(PluginSystemPlugin):
+
+class ModulePlugin(PythonPluginSystemPlugin):
```

Comment:
[question] Do we need to consider moving these python plugins into a separate `python` space as part ofd this too?

Reply:
... So we have a python-only search path, at the point we have multiple plugin systems living in this folder, with their own search tests? That being said, we maybe want to leave it where it is, so we can double check coexistence between the two systems works too!

Reply:
Does that open a can of worms, i.e. should then the `pluginSystem` package have a `python` subpackage?  Maybe we should just do that now, but I'm happy to defer until we work on the C++ plugin system, YAGNI and all that.

Another argument, specific to these tests, is perhaps that any plugin defined in Python must by definition be using the Python plugin system, so no need to disambiguate.

Reply:
Lets save it till we know what it looks like. We will def need too support a shared plugin path for both plugin systems,